test/Integration/Helper.ts

```diff
@@ -16,9 +16,18 @@ export class Client {
     return fetch(this.url + '/works/' + id)
   }
 
+  getWorks() {
+    return fetch(this.url + '/works/')
+  }
+
+  getWorksTest(test?: string) {
```

Comment:
Not really sure this is necessary. But included it just in case.

Reply:
Looks good!